sdk/src/feature_set.rs

```diff
@@ -143,6 +143,10 @@ pub mod add_missing_program_error_mappings {
     solana_sdk::declare_id!("3QEUpjhgPEt92nz3Mqf6pABkHPGCQwSvKtyGMq4SuQyL");
```

Comment:
We would like to retain control over when and where this feature is enabled, please replace the id here with `HTW2pSyErTj4BV6KBM9NZ9VBUJVxt7sacNWcf76wtzb3`